src/pyodide/python.js

```diff
@@ -0,0 +1,107 @@
+import { _createPyodideModule } from "pyodide-internal:pyodide-bundle/pyodide.asm";
```

Comment:
Non-blocking: could/should a lot of the code in here be upstreamed to Pyodide? of course the imports are workerd-specific, but could the snapshot logic/filesystem prep be upstreamed?

Reply:
The filesystem stuff is already present upstream, but it's awkward for us to use it because it's bundled into `pyodide.js` so we'd have to use a custom build. I'm thinking about upstreaming the memory snapshotting, but it'll probably take a while for me to get around to it.